TruePath/AbsoluteFile.cs

```diff
@@ -0,0 +1,72 @@
+// SPDX-FileCopyrightText: 2024 TruePath contributors <https://github.com/ForNeVeR/TruePath>
+//
+// SPDX-License-Identifier: MIT
+
+using System.Runtime.Versioning;
+using System.Text;
+
+namespace TruePath;
+
+public static class AbsoluteFile
```

Comment:
Same design notes as for the `AbsoluteDirectory`: let's merge these two into `public static class FileIO`, and mark all the methods as extensions.